backend/src/main/java/com/bakdata/conquery/models/query/preview/EntityPreviewExecution.java

```diff
@@ -0,0 +1,122 @@
+package com.bakdata.conquery.models.query.preview;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.bakdata.conquery.apiv1.FullExecutionStatus;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.io.storage.MetaStorage;
+import com.bakdata.conquery.models.auth.entities.Subject;
+import com.bakdata.conquery.models.auth.entities.User;
+import com.bakdata.conquery.models.config.ConqueryConfig;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.execution.ManagedExecution;
+import com.bakdata.conquery.models.forms.managed.AbsoluteFormQuery;
+import com.bakdata.conquery.models.forms.managed.ManagedForm;
+import com.bakdata.conquery.models.messages.namespaces.WorkerMessage;
+import com.bakdata.conquery.models.messages.namespaces.specific.ExecuteForm;
+import com.bakdata.conquery.models.query.ColumnDescriptor;
+import com.bakdata.conquery.models.query.ManagedQuery;
+import com.bakdata.conquery.models.query.PrintSettings;
+import com.bakdata.conquery.models.query.SingleTableResult;
+import com.bakdata.conquery.models.query.resultinfo.ResultInfo;
+import com.bakdata.conquery.models.query.results.EntityResult;
+import com.bakdata.conquery.models.query.results.MultilineEntityResult;
+import com.bakdata.conquery.models.worker.DatasetRegistry;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.MoreCollectors;
+import lombok.NonNull;
+
+/**
+ * Dedicated {@link ManagedExecution} to properly display/combine the two Queries submitted by {@link EntityPreviewForm}.
+ * This mostly delegates to {@link EntityPreviewForm#VALUES_QUERY_NAME}, but embeds the result of {@link EntityPreviewForm#INFOS_QUERY_NAME} into {@link EntityPreviewStatus#getInfos()}.
+ */
+@CPSType(id = "ENTITY_PREVIEW_EXECUTION", base = ManagedExecution.class)
+public class EntityPreviewExecution extends ManagedForm implements SingleTableResult {
+
+	@Override
+	public boolean isSystem() {
+		// This Form should NEVER be started manually. Nor persisted
+		return true;
+	}
+
+	EntityPreviewExecution(EntityPreviewForm entityPreviewQuery, User user, Dataset submittedDataset) {
+		super(entityPreviewQuery, user, submittedDataset);
+	}
+
+	/**
+	 * Takes a ManagedQuery, and transforms its result into a List of {@link EntityPreviewStatus.Info}.
+	 * The format of the query is an {@link AbsoluteFormQuery} containing a single line for one person. This should correspond to {@link EntityPreviewForm#VALUES_QUERY_NAME}.
+	 */
+	private List<EntityPreviewStatus.Info> transformQueryResultToInfos(ManagedQuery infoCardExecution, DatasetRegistry datasetRegistry, ConqueryConfig config) {
+		final MultilineEntityResult result = (MultilineEntityResult) infoCardExecution.streamResults().collect(MoreCollectors.onlyElement());
+		final Object[] values = result.getValues().get(0);
```

Comment:
Hier bitte doku, warum es nur element 0 gibt oder nur das interessant ist

Reply:
steht in der Beschreibung, hab es aber nochmal expliziter drüber geschrieben.